src/reference/asciidoc/si-kafka.adoc

```diff
@@ -156,24 +170,27 @@ private KafkaProducerMessageHandlerSpec<Integer, String, ?> kafkaMessageHandler(
             .configureKafkaTemplate(t -> t.id("kafkaTemplate:" + topic));
 }
 ----
+====
 
-If a `send-failure-channel` (`sendFailureChannel`) is provided and a send failure is received (sync or async), an `ErrorMessage` is sent to the channel.
-The payload is a `KafkaSendFailureException` with properties `failedMessage`, `record` (the `ProducerRecord`) and `cause`.
-The `DefaultErrorMessageStrategy` can be overridden via the `error-message-strategy` property.
+If a `send-failure-channel` (`sendFailureChannel`) is provided and a send failure (sync or async) is received, an `ErrorMessage` is sent to the channel.
+The payload is a `KafkaSendFailureException` with `failedMessage`, `record` (the `ProducerRecord`) and `cause` properties.
+You can override the `DefaultErrorMessageStrategy` by setting the `error-message-strategy` property.
 
-If a `send-success-channel` (`sendSuccessChannel`) is provided, a message with a payload of type `org.apache.kafka.clients.producer.RecordMetadata` will be sent after a successful send.
+If a `send-success-channel` (`sendSuccessChannel`) is provided, a message with a payload of type `org.apache.kafka.clients.producer.RecordMetadata` is sent after a successful send.
 
 [[si-inbound]]
-==== Message Driven Channel Adapter
+==== Message-driven Channel Adapter
 
 The `KafkaMessageDrivenChannelAdapter` (`<int-kafka:message-driven-channel-adapter>`) uses a `spring-kafka` `KafkaMessageListenerContainer` or `ConcurrentListenerContainer`.
 
-Starting with _spring-integration-kafka version 2.1_, the `mode` attribute is available (`record` or `batch`, default `record`).
-For `record` mode, each message payload is converted from a single `ConsumerRecord`; for mode `batch` the payload is a list of objects which are converted from all the `ConsumerRecord` s returned by the consumer poll.
-As with the batched `@KafkaListener`, the `KafkaHeaders.RECEIVED_MESSAGE_KEY`, `KafkaHeaders.RECEIVED_PARTITION_ID`, `KafkaHeaders.RECEIVED_TOPIC` and `KafkaHeaders.OFFSET` headers are also lists with, positions corresponding to the position in the payload.
+Starting with spring-integration-kafka version 2.1, the `mode` attribute is available. It can accept values of `record` or `batch` (default: `record`).
```

Comment:
OK. I will fix this chapter on merge: too much inconsistency with "Once sentence per line"